themes/default/content/blog/next-level-iac-pulumi-automation-api/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: "Next-level IaC: Pulumi Automation API and API Economy"
```

Comment:
I think we should drop Automation API from the title. I want people to just focus on the use case or unique mechanism.